components/partials/ArticleFooter.vue

```diff
@@ -1,17 +1,194 @@
+      <div class="article-footer__author-details">
+        <span class="article-footer__author-fullname article-footer__details-text"> {{ author.fullname }} </span>
+        <span class="article-footer__author-description article-footer__details-text">This is Photoshop's version of Lorem Ipsum. Proin gravida nibh vel velit auctor...</span>
+        <span class="article-footer__author-leader-sections">
+          <ul class ="article-footer__section-tags">
+            Lider sekcji:
+            <li v-for="(ltag, index) in leaderTags" class="article-footer__single-leader-tag" :key = "index">
+              <router-link to="" :style="{ 'background-color': ltag.color }" class="article-footer__single-section-tag-route">
+                {{ ltag.text }}
+              </router-link>
+            </li>
+          </ul>
+        </span>
+        <span class="article-footer__author-member-sections">
+          <ul class ="article-footer__section-tags">
+            Członek sekcji:
+            <li v-for="(mtag, index) in memberTags" class="article-footer__single-member-tag" :key = "index">
+              <router-link to="" :style="{ 'background-color': mtag.color }" class="article-footer__single-section-tag-route">
+                {{ mtag.text }}
+              </router-link>
+            </li>
+          </ul>
+        </span>
+      </div>
+      <span class="article-footer__icons">
+        <router-link to="" class="article-footer__single-icon flaticon-like"> {{ likes.length }} </router-link>
+        <router-link to="" class="article-footer__single-icon flaticon-chat"> {{ comments.length }} </router-link>
+        <router-link to="" class="article-footer__single-icon flaticon-facebook-logo"></router-link>
+        <router-link to="" class="article-footer__single-icon flaticon-code-fork"></router-link>
+        <router-link to="" class="article-footer__single-icon flaticon-cog-wheel"></router-link>
+      </span>
+    </div>
   </footer>
 </template>
 
 <script>
-export default {
+import articleFooterSchema from '~/schemes/article/footer'
 
+export default {
+  data () {
+    return {
+      tags: [
+        'Java',
+        'Programowanie',
+        'Kotlin',
+        'Technologia'
+      ],
+      leaderTags: [
+        {
+          text: 'Java',
+          color: '#dd5826'
+        }
+      ],
+      memberTags: [
+        {
+          text: 'Sztuczna Inteligencja',
+          color: '#64bd63'
+        },
+        {
+          text: 'WebDev',
+          color: '#5d8fc2'
+        }
+      ]
+    }
+  },
+  props: articleFooterSchema.props
 }
 </script>
 
 <style lang="scss">
+@import "assets/scss/_imports.scss";
 
-</style>
+.article-footer {
+  padding: 0 30px 30px 30px;
+  height: 250px;
+  background-color: $article-footer-bg-color;
+
+  &__tags {
+    list-style: none;
+    padding-bottom: 20px;
+  }
+
+  &__single-tag {
+    display: inline-block;
+    margin-right: 16px;
+    position: relative;
+    font-size: .8rem;
```

Comment:
Please add hover and focus effect like in article card on the homepage. Just lighter gray color will be ok.